includes/class-sensei-wc.php

```diff
@@ -1233,12 +1233,15 @@ public static function complete_order( $order_id = 0 ) {
 		    return;
         }
 		$order_status = Sensei_WC_Utils::get_order_status( $order );
-		
-		$accepted_order_stati = array( 'wc-completed', 'wc-processing' );
-		
-		$additional_order_stati = apply_filters( 'sensei_wc_add_order_stati', $accepted_order_stati );
 
-		if ( ! in_array( $order_status,  $additional_order_stati ) ) {
+		/**
+		 * Allow order statuses to be filtered.
+		 *
+		 * @since 1.10.1
```

Comment:
This will go in the next release, which is 1.11.0.
1.10.1 has already shipped. Cheers!